src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -281,22 +239,19 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
 
   @Watch('getShowErrors')
   @Watch('minAuthorizationDate') // because Authorization Date depends on this
-  @Watch('authorization.date') // because Expiry Date depends on this
-  @Watch('authorization.expiryDate')
+  @Watch('authorization.date') // in case this changes
```

Comment:
Because we want to re-validate (and update `authorizationDateValid`) if it changes.